tests/suites/test_suite_ssl.data

```diff
@@ -9241,6 +9241,28 @@ SSL TLS 1.3 Key schedule: Derive-Secret( ., "res master", hash)
 # Vector from RFC 8448
 ssl_tls1_3_derive_secret:MBEDTLS_MD_SHA256:"e2d32d4ed66dd37897a0e80c84107503ce58bf8aad4cb55a5002d77ecb890ece":tls1_3_label_res_master:"c3c122e0bd907a4a3ff6112d8fd53dbf89c773d9552e8b6b9d56d361b3a97bf6":32:MBEDTLS_SSL_TLS1_3_CONTEXT_HASHED:"5e95bdf1f89005ea2e9aa0ba85e728e3c19c5fe0c699e3f5bee59faebd0b5406"
 
+SSL TLS 1.3 Key schedule: Early secrets derivation helper
```

Comment:
Add more test cases for each key derivation.

Reply:
Can you elaborate? There's no point in adding tests for the sake of adding tests. If there's a gap, we should aim to fill it, and if there are acknowledged test vectors, we should test them. Is there a gap beyond the external PSK binder one mentioned above?

Reply:
Note also that there are _zero_ tests for the TLS 1.2 key schedule. This isn't meant as an excuse for gaps in the TLS 1.3 code, but should be taken into account when gauging where we put the quality bar for the upstreaming of the TLS 1.3 code.

Reply:
It is a testing paradigm to test the behaviour of a function with at least the kind of inputs it differentiates. Testing for one use case should be strictly avoided. If you meant these test as sanity test and they will be tested with unit or integration tests I am fine with that. If not and they are unit tests then they should be written as black box tests, without knowing how it is implemented, and should be tested thoroughly.

Reply:
@gabor-mezei-arm Can you clarify what you're referring to specifically? Do you mean the hash algorithm? If so, I agree we should add it if there was a public test vector, but in the absence of it, and because the implementation uses the MD interface generically, I'd be inclined to accept this gap.

Reply:
I referred to the hash algorithm. If no other test test vectors are present then I accept it.